suite-native/module-passphrase/src/components/EmptyWalletInfoSheet.tsx

```diff
@@ -0,0 +1,108 @@
+import { useNavigation } from '@react-navigation/native';
+
+import { Icon, IconName } from '@suite-common/icons';
+import { TxKeyPath } from '@suite-native/intl';
+import {
+    AlertBox,
+    BottomSheet,
+    Box,
+    Button,
+    HStack,
+    Pictogram,
+    Text,
+    VStack,
+} from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import {
+    PassphraseStackParamList,
+    PassphraseStackRoutes,
+    RootStackParamList,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+
+type EmptyWalletInfoSheetProps = {
+    onClose: () => void;
+};
+
+const buttonWrapperStyle = prepareNativeStyle(() => ({
+    width: '100%',
+}));
+
+const listItemStyle = prepareNativeStyle(() => ({
+    width: '90%',
+}));
+
+const ListItem = ({
+    iconName,
+    translationKey,
+}: {
+    iconName: IconName;
+    translationKey: TxKeyPath;
+}) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <HStack spacing={20}>
+            <Icon name={iconName} />
+            <Text style={applyStyle(listItemStyle)}>
+                <Translation id={translationKey} />
+            </Text>
+        </HStack>
+    );
+};
+
+type NavigationProp = StackToStackCompositeNavigationProps<
+    PassphraseStackParamList,
+    PassphraseStackRoutes.PassphraseEmptyWallet,
+    RootStackParamList
+>;
+
+export const EmptyWalletInfoSheet = ({ onClose }: EmptyWalletInfoSheetProps) => {
+    const navigation = useNavigation<NavigationProp>();
+
+    const { applyStyle } = useNativeStyles();
+
+    const handleOpenEmptyWallet = () => {
+        navigation.navigate(PassphraseStackRoutes.PassphraseVerifyEmptyWallet);
+        onClose();
+    };
+
+    return (
+        <BottomSheet isVisible={true} onClose={onClose} isCloseDisplayed={false}>
```

Comment:
isVisible should be passed from parent component as a prop.

Reply:
https://github.com/trezor/trezor-suite/pull/12193/commits/fd61a43929e4a65804e39a40a5c1d039432da8cf